### DISCUSSION TITLE
How do you set video options for code outputs? (revealjs)

### DISCUSSION THREAD

**Discussion by agerlach** [quarto-dev/quarto-cli#7970] [Q&A]
I would like to step through fragments in a .r-stack of a series of images and then finally a video that show a build of a process with RevealJS. I need the video to be placed and sized the same as the images. Each image and video are produced via code. 

The images line up fine, but the video gets scaled to the full width. This seems to be the expected behavior according to the docs.

> Videos are automatically rendered responsively using the full width of the document’s main text column

My question is how do you set video size settings when the video is produced via code? I see how to do it if explicitly embedding an video, but not if done via code. Is there a way to turn off full width scaling of video?

I am using v1.4.520

````
---
title: title
jupyter: julia-1.9
format:
  revealjs:
    auto-stretch: false
---

## Heading

:::: {.r-stack}

```{julia}
using GLMakie
set_theme!()
data = rand(10)

f = Figure(size = (800, 600))
ax = Axis(f[1,1], title = "Image")
lines!(ax, data)
f
```

::: {.fragment}

```{julia}
ax.title = "video"
framerate = 24
time_stamps = range(0.0, 5.0; step = 1/framerate)
anim = Record(f, time_stamps) do t
    nothing
end
anim
```
:::
::::
````

**Comment by mcanouil** [comment]
This is not a Quarto question per se.
You need to use HTML/CSS and/or the library you are using to generate the video.
I suggest you read the documentation of the module and/or ask in their repository.
